src/lib.rs

```diff
@@ -1,5 +1,63 @@
+#![forbid(unsafe_code)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
 #![doc = include_str!("../README.md")]
+//! #### Examples
+//!
+//! ```rust
+//! use resend_rs::{Client, Result};
+//!
+//! #[tokio::main]
+//! async fn main() -> Result<()> {
+//! //     let resend = Resend::default();
+//! //     let _ = resend.emails.send().await?;
+//!
+//!     Ok(())
+//! }
+//! ```
```

Comment:
This is slightly outdated, we could use something like this:

```
#![forbid(unsafe_code)]
#![cfg_attr(docsrs, feature(doc_cfg))]
#![doc = include_str!("../README.md")]
//! #### Examples
//!
//! ```rust,no_run
//! use resend_rs::{types::SendEmailRequest, Client, Result};
//! 
//! #[tokio::main]
//! async fn main() -> Result<()> {
//!     let resend = Client::default();
//! 
//!     let req = SendEmailRequest::new(
//!         "rss2email@resend.dev".to_string(),
//!         vec!["antonis.barotsis@gmail.com".to_string()],
//!         "subject".to_string(),
//!     )
//!     .with_text("Hello world!");
//! 
//!     let res = resend.emails.send(req).await?;
//! 
//!     println!("{}", res.id);
//! 
//!     Ok(())
//! }
//! ```
```

Feel free to make any changes to this. It might also be a good idea to make this change at the very end in case we make any relevant changes in the meantime.

Edit: I just now realized I almost rewrote the test you made in [`email.rs`](https://github.com/martsokha/resend-rs/blob/6e7b2319bd37a25b71b55d9e2165a7e546e3b5d7/src/emails.rs#L351-L371), feel free to use that instead, just remember the `no_run`.

Reply:
Removed test from `email.rs`, moved it here with `no_run`.